### ISSUE TITLE
Graph code - Graph.tsx error (Mac user, using Sublime Text)

### ISSUE THREAD

**Issue by rb-ctrl** [insidesherpa/JPMC-tech-task-2-PY3#394]
Can anybody spot the error and know of how to resolve this? 

- Below is the error message I receive on Google Chrome when trying to display the graph:

![Screenshot 2020-06-24 at 13 58 43](https://user-images.githubusercontent.com/67052787/85561301-5dadc000-b623-11ea-9b49-4d2ce9867419.png)

- Below is the code outlining the error:

![Screenshot 2020-06-24 at 13 59 28](https://user-images.githubusercontent.com/67052787/85561389-73bb8080-b623-11ea-9f90-6183de58af09.png)

- Below is the code from the file 'Graph.tsx' in Sublime Text application:

![Screenshot 2020-06-24 at 12 35 16](https://user-images.githubusercontent.com/67052787/85561187-466ed280-b623-11ea-88c7-3171e7966d89.png)

**Can anybody spot the error and know of how to resolve this? :)**

**Comment by harbhajansharma04** [comment]
same error occurred

**Comment by sakshikatolkar** [comment]
check #233 
it worked for me.

**Comment by rb-ctrl** [comment]
> check #233
> it worked for me.

Sadly, #233 didn't resolve the issue...

**Comment by rb-ctrl** [comment]
I have managed to solve this issue!! 

There were some mistakes in my Graph.tsx file. I have attached screenshots of my correct code. I hope this helps! :) 

![Screenshot 2020-07-29 at 16 04 59](https://user-images.githubusercontent.com/67052787/88817373-585d0b80-d1b5-11ea-990d-0de73de4ba47.png)
![Screenshot 2020-07-29 at 16 05 11](https://user-images.githubusercontent.com/67052787/88817382-5a26cf00-d1b5-11ea-8001-ebc663957317.png)
